sdc/datatypes/hpat_pandas_series_rolling_functions.py

```diff
@@ -355,6 +381,13 @@ def impl(self):
         length = len(input_arr)
         output_arr = numpy.empty(length, dtype=float64)
 
+        if minp == 0 and win == 0:
+            for i in prange(length):
+                output_arr[i] = init_result
```

Comment:
I'm not sure... shouldn't it be something like:
```python
output_arr[i] = get_result(nfinite, minp, init_result)
```

@densmirn

Reply:
BTW is it correct behavior at all - create rolling with minp == 0 and win == 0?

Reply:
@AlexanderKalistratov it's correct from Pandas side, but strange in general.

Reply:
@Rubtsowa the extension looks strange. The old code didn't have such checks.

Reply:
Do we have tests for this, btw?

Reply:
@AlexanderKalistratov yes:
```python
for window in range(0, len(obj) + 3, 2):
    for min_periods in range(0, window + 1, 2):
        # ...
```

Reply:
@Rubtsowa so, count fails this tests without the fix?